src/app/components/Persist/LoginModalLayout.tsx

```diff
@@ -1,26 +1,43 @@
 import { Box } from 'grommet/es6/components/Box'
 import { ResponsiveLayer } from '../ResponsiveLayer'
-import React from 'react'
+import React, { useContext } from 'react'
 import { Header } from 'app/components/Header'
 import { Navigation } from '../Sidebar'
+import walletBlueLogotype from '../../../../public/Rose Wallet Logo Blue cropped.svg'
+import walletWhiteLogotype from '../../../../public/Rose Wallet Logo White cropped.svg'
+import { ThemeContext } from 'styled-components'
 
 export function LoginModalLayout(props: {
   title: string
   children: React.ReactNode
   onClickOutside?: () => void
   onEsc?: () => void
 }) {
+  const { dark } = useContext<any>(ThemeContext)
+
   return (
     <Box direction="row-responsive" background="background-back" fill style={{ minHeight: '100dvh' }}>
       <Navigation />
       <ResponsiveLayer
         modal
+        responsive={false}
+        style={{
+          width: 'min(100%, 500px)',
```

Comment:
Can we avoid inline styles, and use styled-components instead?

Also don't see how this is different from:

```css
width: 100%
max-width: 500px;
```

Make is much easier to read.

Reply:
Inline styles:
I would refactor if it's reusable

max-width:
Oh, yep, that does work
(I started from https://github.com/oasisprotocol/oasis-wallet-web/blob/ab5675f07b50e06ba2511d55bf6c9e7c7c98df20/src/app/pages/ParaTimesPage/ParaTimeContent/index.tsx#L23 but that doesn't work correctly in this case)